packages/cloud_firestore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.6
```

Comment:
0.9.5+2 since it is pre 1.0.0
https://www.dartlang.org/tools/pub/versioning#semantic-versions

Reply:
👍